stream/classes/stages/InterCoreMappingStage.py

```diff
@@ -25,18 +32,18 @@ class InterCoreMappingStage(Stage):
 
     def __init__(
         self,
-        list_of_callables,
+        list_of_callables: list[StageCallable],
         *,
-        workload,
-        accelerator,
-        node_hw_performances,
-        nb_ga_generations,
-        nb_ga_individuals,
-        plot_hof,
-        plot_file_name,
-        plot_full_schedule=False,
-        plot_data_transfer=False,
-        operands_to_prefetch,
+        workload: Workload,
```

Comment:
Should this be a `Workload`? Is it not a `DiGraph`? from my understanding, after the workload is split into computational nodes, it stops being a `Workload` and it becomes `DiGraph`.

Reply:
Aaah, thank you for the clarification!